pynocchio/pynocchio.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         self.setStyle('Fusion')
         try:
             self.setStyleSheet(qdarkgraystyle.load_stylesheet())
-        except ImportError:
+        except NameError:
```

Comment:
@Jellby thanks for the fiz. Please, change the message to something more descriptive like: "Fix Exception class in import theme function"

Reply:
Is it OK to "rebase" (i.e., rewrite commits that have already been pushed)?

Reply:
Yes, this commit: https://github.com/mstuttgart/pynocchio/pull/89/commits/083b17b9178b194aa0b66d33b0aeb8fb669233bd